src/det.jl

```diff
@@ -0,0 +1,8 @@
+function det(M::Matrix{<:AbstractPolynomialLike})
```

Comment:
I tend to prefer explicitely writing Compat.LinearAlgebra.det here and not doing the import. That is more readable as the information that is is tha Base det function and not a new one is not split in two places.
What do you think ?

Reply:
@blegat Sorry I merged a little bit too fast here. I addressed this in #87

Reply:
@saschatimme no problem ;)